resource/plugins/lv_icon.lvproj

```diff
@@ -0,0 +1,568 @@
+﻿<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
The project file was moved to the top level. If there were changes to the project file, they should be applied there.